packages/workbox-build/src/lib/errors.js

```diff
@@ -45,8 +45,8 @@ module.exports = {
   'sw-write-failure': 'Unable to write the service worker file.',
   'sw-write-failure-directory': ol`Unable to write the service worker file;
     'swDest' should be a full path to the file, not a path to a directory.`,
-  'unable-to-copy-workbox-sw': ol`workbox-sw is needed by the service worker
-    and could not be copied over to your new site.`,
+  'unable-to-copy-workbox-libraries': ol`One or more libraries needed by your
```

Comment:
I might change the wording of this to make it sound more like a workbox problem. This sounds like it could be the developers problem. 

"One of more of the Workbox librares could not be copied over to '${destDirectory}'"

Reply:
Updated.